docs/SECURITY.md

```diff
@@ -1,12 +1,30 @@
 # Security Policy
 
+The code-server team (and Coder, the organization) care a lot about keeping the project secure and safe for end-users.
+
+## Tools
+
+We use a combination of tools to help us stay up-to-date and fix vulnerabilities.
+
+- [dependabot](https://dependabot.com/)
+  - Automatically keeps code-server up-to-date by updating dependencies in response to security alerts.
+  - See alerts [here](https://github.com/cdr/code-server/security/dependabot).
```

Comment:
suggest avoiding links to "here" as they are less accessible (screen readers won't read them correctly), and the short links are harder to press on a phone. Also, non-maintainers won't be able to see that page (try opening it in an Incognito window) so it doesn't seem useful to link to it

Reply:
Ah good point about screen readers - forgot about that. And totally thought they were open to non-maintainers 😬

Good call. I'll remove.